src/components/lowering_replay.js

```diff
@@ -121,7 +122,17 @@ class LoweringReplay extends Component {
       let loweringStartTime = moment(this.state.lowering.start_ts);
       let loweringNow = moment(this.props.event.events[v].ts);
       let loweringElapse = loweringNow.diff(loweringStartTime);
-      return moment.duration(loweringElapse).format("d [days] hh:mm:ss");
+
+      if (this.state.utcScrubber){
+        return moment(loweringNow).toISOString(); // formats scrubber tooltip in UTC ISO8601
+      }
+      else{
+        return moment.duration(loweringElapse).format("d [days] hh:mm:ss");
```

Comment:
The code formatting is not *super* consistent, but I try to match it best I can, e.g., with a space between the `) {`.

You can reduce [cyclomatic complexity](https://en.wikipedia.org/wiki/Cyclomatic_complexity) and make it easier to find the final `return` statement by writing:

```
if (condition) {
    return x;
}

return y;
```